src/components/Picker-group/Picker-group.js

```diff
@@ -10,31 +10,43 @@ export default class PickerGroup extends React.Component {
   render() {
     const uuid = uuid4();
```

Comment:
Is the 2+ Picker Groups issue fixed?

Reply:
not yet will work on it in another branch